Tests/FluentAssertions.Specs/Execution/AssertionScope.FormattingOptionsSpecs.cs

```diff
@@ -0,0 +1,128 @@
+using System;
+using FluentAssertions.Execution;
+using Xunit;
+
+namespace FluentAssertions.Specs.Execution;
+
+[Collection("FormatterSpecs")]
```

Comment:
Reason for this attribute: I faced issues with several other unrelated tests (mostly inside `FormattingSpecs`)  failing, when this is omitted.

Reply:
Since this solution does not fit my personal requirements, I pushed an alternative... 

Please let me know what you would prefer